src/com/sap/piper/ConfigurationHelper.groovy

```diff
@@ -47,6 +47,18 @@ class ConfigurationHelper implements Serializable {
         return this
     }
 
+    ConfigurationHelper handleCompatibility(Script script, Map compatibleParameters){
```

Comment:
rename `script` to `step`, we used `script` to refer to the `Jenkinsfile` in the library.